core/native/src/test/scala/com/softwaremill/sttp/testing/SyncHttpTest.scala

```diff
@@ -0,0 +1,372 @@
+package com.softwaremill.sttp.testing
+
+import java.io.ByteArrayInputStream
+import java.nio.ByteBuffer
+
+import com.softwaremill.sttp.internal.SttpFile
+import com.softwaremill.sttp._
+import org.scalatest._
+
+import scala.concurrent.duration._
+
+trait SyncHttpTest
```

Comment:
If this test stays as-is (copy-pasted), maybe you could add a comment explaining why it's copied and how to add how the tests cases differ from `HttpTest`